src/core/qgsmaplayerstylemanager.h

```diff
@@ -161,6 +161,7 @@ class CORE_EXPORT QgsMapLayerStyleManager : public QObject
     QMap<QString, QgsMapLayerStyle> mStyles;
     QString mCurrentStyle;
     QgsMapLayerStyle *mOverriddenOriginalStyle = nullptr;
+    QString mEmptyStyleName = tr( "default" );
```

Comment:
SOrry, I read too quick ;-)
I thought that every private members/methods should have a `m` prefix. I will fix that